model/account_payment_model.go

```diff
@@ -577,6 +723,61 @@ func PlanPaymentsWithConfig(ctx context.Context, subsidyConfig *SubsidyConfig) (
 	return
 }
 
+func payoutChildrenReferralNetworks(
```

Comment:
It's usually safer to use stack/list for graph traversal to avoid any issues with call stack size. This might not be an issue on Go tho with the way Go manages the call stack.